gtoc/gtoc.go

```diff
@@ -0,0 +1,66 @@
+// Package gtoc contains the logic to convert a GOBL envelope into a CII document
+package gtoc
+
+import (
+	"fmt"
+
+	"github.com/invopop/gobl"
+	"github.com/invopop/gobl.cii/document"
+	"github.com/invopop/gobl/bill"
+)
+
+// Converter is the struct that contains the logic to convert a GOBL envelope into a CII document
+type Converter struct {
+	doc *document.Document
+}
+
+// NewConverter Builder function
+func NewConverter() *Converter {
+	c := new(Converter)
+	c.doc = new(document.Document)
+	return c
+}
+
+// GetDocument returns the CII document
+func (c *Converter) GetDocument() *document.Document {
+	return c.doc
+}
+
+// ConvertToCII converts a GOBL envelope into a CIIdocument
+func (c *Converter) ConvertToCII(env *gobl.Envelope) (*document.Document, error) {
```

Comment:
is `ToCII` necessary? The entry method here could just be `Convert`, without the model behind the scenes.